index-handlers/src/test/java/no/sikt/nva/nvi/index/OpenSearchClientTest.java

```diff
@@ -294,15 +294,17 @@ void shouldReturnSingleDocumentWhenFilteringByYear() throws InterruptedException
     }
 
     @Test
-    void shouldReturnSingleDocumentWhenFilteringByTitle() throws InterruptedException, IOException {
+    void shouldReturnSingleDocumentWhenFilteringBySearchTermAndTitle() throws InterruptedException, IOException {
```

Comment:
I think I would test SearchTerm in a separate test to test one functionality in one test. `shouldReturnSingleDocumentWhenSearchingForTerm()` or something.